section2-week1/exercise7/Main.cpp

```diff
@@ -11,93 +11,96 @@ using namespace std;
 
 int main()
 {
-	int value, random, answer;
-	ans = 0;
+	int value, random;
 
-	srand(time(NULL));
-	random = rand() % (1 - 4);
 
 	cout << "Hey you, let's play Rock, Paper, Scissors!" << endl;
-
 	cout << "1 - Rock\n2 - Paper\n3 - Scissors" << endl;
 	cout << "choose you weapon: " << endl;
 
-	cin >> value;
+	enum Weapon
+	{
+		Rock, Paper, Scissors
+	};
+	srand(time(NULL));
+	random = rand() % (1 - 4);
+	Weapon weapon = (Weapon)random;
 
-	cout << "\nI have selected the weapon: " << random << endl;
+	cin >> value;
 
 
+	enum Answers
```

Comment:
Enums shouldn't really be named in plural. It may be a list of answers, but the actual variable type won't be. So for example, later you have the line `Answers answer = tie;` Which is weird that a single answer is of a type called Answers.